src/components/formio/radio.stories.ts

```diff
@@ -88,3 +88,36 @@ export const WithErrors: Story = {
     await expect(canvas.queryByText('Example error')).toBeInTheDocument();
   },
 };
+
+export const DefaultValueRemoved: Story = {
+  args: {
+    label: 'With removing the default value',
+    options: [
+      {value: 'a', label: 'A'},
+      {value: 'b', label: 'B'},
+    ],
+  },
+
+  parameters: {
+    formik: {
+      initialValues: {'my-radio': 'a'},
+    },
+  },
+
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+    const input1 = canvas.getByLabelText('A');
+    const input2 = canvas.getByLabelText('B');
+
+    // Before clearing the default value
+    await expect(canvas.queryByText('Clear selection')).toBeInTheDocument();
```

Comment:
can't this be the same `getByRole('button')` query? it's more specific than just finding the text in the document and you use it anyway to interact with it